aspnetcore/fundamentals/middleware/sample/Chain/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.1",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
```

Comment:
This is the only thing I didn't change. I can't get **anything** to work with 1.1 after installing VS 2017 RC on my machine. (Even tested on a couple of VMs). I did remove `Diagnostics` as it's not used.